src/Api/Teams.php

```diff
@@ -16,6 +16,53 @@ public function all()
         return $this->get('/teams/');
     }
 
+    public function create($name, $canEditTeamPackages = false, $canAddPackages = false, $canCreateSubrepositories = false, $canViewVendorCustomers = false, $canManageVendorCustomers = false)
```

Comment:
I am unsure about the method signature for create/edit. This might end up really confusing if we add more permissions and their order doesn't make sense anymore. I wonder if having a TeamPermissions object with public properties might be easier?

Reply:
see slack. Not yet sold on the flags in the object yet. But I would adjust to the majority of oppinions